iris/meta.yaml

```diff
@@ -20,14 +20,13 @@ requirements:
         - biggus >=0.14
         - cartopy >=0.14
         - netcdf4
-        - numpy x.x
+        - numpy
```

Comment:
yes, this is no longer required, as i have removed the pp_packing build step, which is obsolete and deprecated, replaced by mo_pack